### ISSUE TITLE
Should mentor retrospectives be stored in the master reference?

### ISSUE THREAD

**Issue by shiryz** [foundersandcoders/master-reference#733]
Every week the mentors have a retrospective with the cf (we're doing that Nazareth and I got the impression @astroash is doing the same), in previous cohorts we stored the minutes and notes in the mr, while these notes are useful for mentor handover they are never used again afterwards.
My suggestion is:
- during the week, the cf creates a hackmd for mentors to add notes to ([template](https://hackmd.io/EzCsDMAZ3BTBaUAjF8AsAOUp4E4AmSAhuvsAIwDMA7EbmqLKEA==) I'm using).
- on the last day of the week review the notes and add any that are missing.
- send the notes to other cfs so they can share with the mentors.
- raise issues when necessary.
  
@ZooeyMiller, @astroash (or anyone else) what do you guys think?

**Comment by eliascodes** [comment]
@shiryz where are these retrospectives? I can't find them.

In principle, I agree that these things shouldn't go into the MR raw. The CF's job (IMO) is to determine together with the mentors which things arising from the week need to be communicated to the MR via issues or documentation.

**Comment by astroash** [comment]
Yeah, I agree with @eliascodes. I think it's very useful to go through the process of retrospectives as with a part-time CF role you cannot be around for the full week, so it's useful to hear a week overview from them. Then you can raise issues in the relevant places/ encourage them to, so improvements can be documented. I worry if they're documented it could add to _hoarding_ of unread documents, which makes the valuable files harder to find.

**Comment by shiryz** [comment]
I would suggest creating a file in the facilitation folder as a guide for future cfs, following the same format would be really helpful.

**Comment by astroash** [comment]
@shiryz are you suggesting adding a template to this folder for future reference or all of the filled out retrospectives?

**Comment by shiryz** [comment]
@astroash yes + what to do with these notes, I'm thinking we should have a guide in general, not just for retros, but this would be a good place to start from.

**Comment by astroash** [comment]
I think that's a great idea. At the minute I'm copy and pasting the week overview from master reference and then deleting the workshop links so I only have the headings. It works as a nice structure, but not necessarily templatable as the curriculum is ever changing. I also think it should mention it's a _suggested_ template, as retrospectives aren't strictly in the CF description and I think each individual should be given freedom to allocate their time dependant on the needs of their campus and cohort.

How are you structuring yours?

**Comment by shiryz** [comment]
@astroash I'm using a similar format but kept the links :sweat_smile: ([example](https://hackmd.io/JwUwTALAJgzBIFooFYBsYETAdgAwIEMxgBGBZKA9A5YAYxICNsg=)).
Agree with the fact that this up to the cf to decide whether or not it's the best use of their time :+1:.

**Comment by mattlub** [comment]
for what it's worth, I think they were stored in the MR because there was a lot of curriculum recommendations/comments in there (that's pretty much all there was right?). I don't know if anyone actually used these in the curriculum planning but that's what I imagine they would be most useful for.

**Comment by shiryz** [comment]
@mattlub yeah that's the idea, do you think it's better in the form of minutes instead of issues?

**Comment by mattlub** [comment]
@shiryz I don't know if anyone actually referred to the mentor retrospectives when suggesting curriculum changes- if they didn't, then I guess having them in the form of minutes was not particularly successful, and raising issues might be more useful/effective. 

I'm a firm believer that the mentors have pretty good opinions on how well the stuff they just delivered went, and I know the idea of mentors being responsible for curriculum updates was discussed at one point (maybe raised by dan)- but I can't remember what others had to say about it.

**Comment by zurda** [comment]
@shiryz , @mattlub , @astroash , while mentoring FAC-16 we've been putting our retrospective notes as an [issue](https://github.com/foundersandcoders/master-reference/issues/933). 

Our process was:
1. While on FAC-15 we reviewed the content of the week we'll be mentoring and discussed it within our cohort
2. Each mentoring pair created an issue in master reference, assigned themselves, and listed their ideas and suggestions for their mentoring week 
3. We then created PR's and linked to that issue to give context to why we created these PR's
4. Lastly, after our mentoring week was over we listed our conclusions in this template: 

> ## Reflections on how the week went: 
> ## Any changes we did + Links to the PRs:
> ## Feedback from students: 
> ## Suggested actions: 

5. Once the retrospective was listed - we assign the CF and un assigned ourselves

This worked great for us (at least I hope - @charlielafosse can share more :) )

**Comment by charlielafosse** [comment]
this is now document in the master ref! @arrested-developer did a lot of great work on documenting this process - and I recently added some updates too. [See here](https://github.com/foundersandcoders/master-reference/blob/master/curriculum-planning/curriculum-update-process.md) for the documentation.
